src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -77,19 +78,29 @@ class WorkspaceReimburseView extends React.Component {
     }
 
     getRateDisplayValue(value) {
-        const numValue = parseFloat(value);
+        const numValue = this.getNumericValue(value);
```

Comment:
Why do we need this change?

Reply:
> Why do we need this change?

`parseFloat` would fail if we pass a number with a comma instead of the dot in it, so we need to replace the decimal separator in `this.getNumericValue` before passing it to `parseFloat`